Pinta.Core/Effects/BaseEffect.cs

```diff
@@ -37,6 +37,12 @@ namespace Pinta.Core;
 [Mono.Addins.TypeExtensionPoint]
 public abstract class BaseEffect
 {
+	/// <summary>
+	/// If the ROIs are subdivided (in rows, for example) and the effect is applied to each subdivision
```

Comment:
I would word this as something like:
```
Specifies whether Render() can be called separately (and possibly in parallel) for different sub-regions of the image.
If false, Render() will be called once with the entire region the effect is applied to.
This is required for effects which cannot be applied independently to each pixel, e.g. if the effect accumulates information from previously processed pixels.
```

Reply:
@cameronwhite I've now used your wording.